runbotics/runbotics-orchestrator-ui/src/utils/text.ts

```diff
@@ -5,8 +5,15 @@ interface CapitalizeFirstLetterParams {
     join?: string
 }
 
-export const capitalizeFirstLetter = ({ text, lowerCaseRest = false, delimiter, join }: CapitalizeFirstLetterParams) =>
-    text
-        .split(delimiter)
-        .map((word) => word.charAt(0).toUpperCase() + (lowerCaseRest ? word.slice(1).toLowerCase() : word.slice(1)))
-        .join(join ?? '');
+export const capitalizeFirstLetter = ({
+    text,
+    lowerCaseRest = false,
+    delimiter,
+    join,
+}: CapitalizeFirstLetterParams) =>
+    !text
```

Comment:
let's go with `text.length === 0` + I believe that if there is possibility for any string then use `trim` also to this condition

Reply:
done, I hope correctly